quick_purchase/views/product_view.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp><data>
+
+    <record id="product_product_tree_view" model="ir.ui.view">
+        <field name="model">product.product</field>
+        <field eval="20" name="priority"/>
+        <field name="arch" type="xml">
+            <tree string="Products" create="0" editable="top">
+                <field name="lst_price" readonly="1"/>
+                <field name="categ_id" readonly="1"/>
+                <field name="default_code" readonly="1"/>
+                <field name="name" readonly="1"/>
+                <field name="qty_to_purchase"/>
```

Comment:
Please could you add this field `<field name="attribute_value_ids" widget="many2many_tags"/>` on the view
because if you use variant, you can need it to distinguish products